paimon-core/src/main/java/org/apache/paimon/schema/TableSchema.java

```diff
@@ -271,6 +274,19 @@ public TableSchema copy(Map<String, String> newOptions) {
                 timeMillis);
     }
 
+    public static TableSchema fromJson(String json) {
+        return JsonSerdeUtil.fromJson(json, TableSchema.class);
+    }
+
+    public static TableSchema fromPath(FileIO fileIO, Path path) {
```

Comment:
What's this used for?

Reply:
It is used in merge branch, so I temporarily deleted it.